### PR TITLE
Updated enrollments that support notifications

### PR DESCRIPTION
**Pull request by maholdaa**
Added details to which enrollment types are and are not supported based on feedback

### CODE REVIEW

**Comment by traya1** [comment]
Learn Build status updates of commit _[bb32f48](https://github.com/maholdaa/memdocs/commits/bb32f4867f18f6fddcdc691eee7a10d7ad57c949)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/enrollment/enrollment-notifications.md](https://github.com/maholdaa/memdocs/blob/patch-3/memdocs/intune/enrollment/enrollment-notifications.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212212351196015-3473/BuildReport?accessString=6a55dcc87d70ed124591acdcda80d3b17d522f01861e2a5979afa7cc599252d0).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Review by lenewsad — CHANGES_REQUESTED** [review]
Hi @maholdaa, thanks for your contribution. What does the "DEP" acronym next to Android stand for?

File: memdocs/intune/enrollment/enrollment-notifications.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ Intune delivers enrollment notifications via email or push notification. You can
 
 Enrollment notifications work on devices running:  
 
-* Android  
-* iOS/iPadOS
+* Android  (excludes DEP scenarios)
+* iOS/iPadOS (excludes DEP or ADE scenarios) 
 * macOS 
-* Windows 10/11 
+* Windows 10/11 (excludes hybrid AADJ)
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/enrollment-notifications.md:48]:
```suggestion
* Windows 10/11 (excludes hybrid Azure AD-joined devices)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @maholdaa, kindly commit the suggested change and expand the acronyms DEP and ADE.

**Comment by lenewsad** [comment]
Hi @maholdaa, following up. There isn't a "DEP" associated w/Android. Can you please expand on what you mean by that acronym? 

For IOS, are we still supporting enrollments set up through "DEP" or were all customers required to upgrade to Apple Business Manager and Apple School Manager? source: https://support.apple.com/en-us/HT209617

**Comment by yogkumgit** [comment]
@maholdaa Gentle reminder. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by maholdaa** [comment]
@lenewsad sorry Laura seeing this now --- maybe instead we can rephrase -- right now for both Apple and Android platform enrollment notifications work for BYOD or personally enrolled scenarios.

**Comment by lenewsad** [comment]
Thanks for confirming @maholdaa!

**Comment by lenewsad** [comment]
#sign-off

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Pull request merged**